### DISCUSSION TITLE
Is there any way to close red_knot just after parsing file, without running mailoop?

### DISCUSSION THREAD

**Discussion by qarmin** [astral-sh/ruff#11930] [General]
Currently I use timeout, to close it after a while, which is not ideal solution

**Comment by MichaReiser** [comment]
No. This isn't supported yet. But red knots isn't really meant to be used just yet and I also don't think it's worth setting up fuzzing just yet. There are simply too many unhandled languages features. 

What are you trying to use it for?

**Reply by qarmin** [reply]
Fuzzing :)
